bouncer/authentication/authentication.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import User
+from rest_framework import authentication
+from rest_framework import exceptions
+from authentication import models
+
+
+class Authentication(authentication.TokenAuthentication):
+    def authenticate(self, request):
```

Comment:
This seems to be exactly the same, as the parent class' (`TokenAuthentication`) method. In that case, it should be completely removed.